metricbeat/module/redis/info/info.go

```diff
@@ -27,9 +27,11 @@ import (
 	"github.com/elastic/beats/metricbeat/module/redis"
 )
 
+var hostParser = parse.URLHostParserBuilder{DefaultScheme: "redis"}.Build()
```

Comment:
What about adding the default port of 6379?

Reply:
`URLHostParserBuilder` doesn't have a default port, I'd leave this for a future change.

Reply:
Yeah, +1 on future change!